benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -19882,8 +19882,8 @@ exports[`Render Summary for multi-table-only-columns.table.html 1`] = `
        >
         <div
          className="cell"
-         dangerouslySetInnerHTML={".000"}
-         title="avg: 0.000, max: 0.000, median: 0.000, min: 0.000, stdev: 0.000"
+         dangerouslySetInnerHTML={"&#x2008;&#x2007;&#x2007;&#x2007;&#x2007;"}
```

Comment:
A sum of `0` values should be represented as `0`, right?

Reply:
Is the question about `0` vs. `.000`? Would depend on the circumstances. For type measure, the latter would also be natural.